nucypher/blockchain/eth/policies.py

```diff
@@ -210,5 +199,14 @@ def make_arrangements(self, network_middleware: RestMiddleware,
             if len(accepted) < self.n:
                 raise Exception("Selected Ursulas rejected too many arrangements")  # TODO: Better exception
 
-        self._accepted_arrangements.update(accepted)
-        self._rejected_arrangements.append(rejected)
+    def publish(self, network_middleware) -> None:
```

Comment:
This is a public interface separate from enact?  This means that policies can be enacted over REST, but not published to the blockchain. Can duplicate arrangements possibly be generated? When will users want enact but not publish?